dat/CMakeLists.txt

```diff
@@ -0,0 +1,276 @@
+set(SPECIAL_LEVELS
+    bigroom.des
+    castle.des
+    endgame.des
+    gehennom.des
+    knox.des
+    medusa.des
+    mines.des
+    oracle.des
+    sokoban.des
+    tower.des
+    yendor.des)
+list(TRANSFORM SPECIAL_LEVELS PREPEND ${NLE_DAT}/)
+
+set(QUEST_LEVELS
+    Arch.des
+    Barb.des
+    Caveman.des
+    Healer.des
+    Knight.des
+    Monk.des
+    Priest.des
+    Ranger.des
+    Rogue.des
+    Samurai.des
+    Tourist.des
+    Valkyrie.des
+    Wizard.des)
+list(TRANSFORM QUEST_LEVELS PREPEND ${NLE_DAT}/)
+
+add_custom_command(
+  DEPENDS makedefs ${NLE_DAT}/bogusmon.txt ${NLE_DAT}/engrave.txt
```

Comment:
What's NLE_DAT doing here? Shouldn't this "just work" with e.g. `bogusmon.txt` b/c it's in the same folder?

Reply:
Sorry, I still had to refactor this (and forgot to update the todo list).

This version of the code sort of relies on BINARY_DIR to be a subdirectory of SOURCE_DIR (but I was a little bit more defensive in writing the paths for the dependencies). In insight, I should be fixing the original problem. Doing this now.

Reply:
Oh wait, if you are referring to -- particularly -- the depends lines, no, that's with respect to BINARY_DIR (so we are in e.g. `build/dat`).